test/08_api-keys.ts

```diff
@@ -20,21 +20,32 @@ describe('create provisioning apikey', function () {
 	[
 		{
 			title: `using /api-key/application/:appId/provisioning endpoint`,
-			fn(user: UserObjectParam | undefined, applicationId: number) {
-				return supertest(user).post(
-					`/api-key/application/${applicationId}/provisioning`,
-				);
+			fn(
+				user: UserObjectParam | undefined,
+				applicationId: number,
+				provisioningKeyName?: string,
+			) {
+				return supertest(user)
+					.post(`/api-key/application/${applicationId}/provisioning`)
+					.send({
+						provisioningKeyName,
+					});
 			},
 		},
 		{
 			title: 'using the /api-key/v1/ endpoint',
-			fn(user: UserObjectParam | undefined, applicationId: number) {
+			fn(
+				user: UserObjectParam | undefined,
+				applicationId: number,
+				provisioningKeyName?: string,
+			) {
 				return supertest(user)
 					.post(`/api-key/v1/`)
 					.send({
 						actorType: 'application',
 						actorTypeId: applicationId,
 						roles: ['provisioning-api-key'],
+						name: provisioningKeyName,
```

Comment:
Let's also test that the provided name is actually set in DB.
I expect this to fail atm actually, since in the code above you are using `{ name: req.body.provisioningKeyName }`.

Reply:
This is already existing endpoint - so will not fail the request. updating to use the `name` value instead of `provisioningKeyName`.

Reply:
> Let's also test that the provided name is actually set in DB.

I mean similarly to how you did it in the SDK  tests.